inc/commondbtm.class.php

```diff
@@ -3844,6 +3845,7 @@ function filterValues($display = true) {
                      if (!class_exists($value)) {
                         $unset = true;
                      }
+                     //go threw plugins
```

Comment:
Not sure.

Reply:
IMHO, we should break here.
```suggestion
                     break;
```

I guess lack of break here is due to a bad code design when this case was added.
https://github.com/glpi-project/glpi/commit/f4f1b0425e3fb9e59e26c90825944184a96308d3#diff-bf93ba44eb9d93a8ce19e63a5b53377d4563ea9df80219a468dffcf0ffe94abfR2389